tests/suites/cloud/tests/ssh-auth/index.js

```diff
@@ -178,9 +172,10 @@ module.exports = {
 		{
 			title: 'SSH authentication in development mode',
 			run: async function(test) {
-				await fse.ensureDir(dirname(sshPath));
+				const customSshPath = join(homedir(), 'custom_id')
+				await fse.ensureDir(dirname(customSshPath));
 				const customKey = await keygen({
```

Comment:
could we use createSSHKey here instead to clean up the code a little bit? This would also make the subsequent ssh-add calls not needed and downsize the code

Reply:
@floion @alexgg It could work, but I'm wondering if using `createSSHKey` is usable in this test case as it will add the key to the ssh-agent https://github.com/balena-os/leviathan/blob/2d9619c9fdb2cbf2fe3e210ea6e9eee19f78b98a/core/lib/common/utils.js#L132

And from what I understand in this part of the test we check that phony keys don't work

Reply:
Right, makes sense :+1: